### PR TITLE
RDS update operation

### PR DESCRIPTION
**Pull request by mdelaossa**
https://app.shortcut.com/aptible/story/9283/support-updating-awspostgres

### CODE REVIEW

File: internal/provider/postgres/resource.go [hunk 1]
```diff
@@ -87,26 +88,60 @@ func resourcePostgresUpdate(ctx context.Context, d *schema.ResourceData, m any)
...
+
+	out, err := c.UpdatePostgres(ctx, input)
 	if err != nil {
 		return diag.FromErr(err)
 	}
 
 	_ = d.Set("environment", out.EnvironmentId)
 	_ = d.Set("id", out.Id)
 	_ = d.Set("handle", out.Handle)
+	_ = d.Set("endpoint", out.Endpoint)
```
Comment (madhuravius) [inline internal/provider/postgres/resource.go:128]: just want to be careful with these and check when running locally that you aren't getting mismatched state (it'll appear in warnings usually)

**Review by madhuravius — COMMENTED** [review]
LGTM - i think the only thing to do is ensure ALL fields stored in backend are hydrated here (so imports will work properly) for the reads

**Comment by mdelaossa** [comment]
@madhuravius @aguilinger sorry for the delay here, this is now ready for review

Something I found out and that you should probably keep in mind is that on `bool` fields, `omitempty` will omit the field if it is set to `false`, which we don't want, so I've removed `omitempty` for my bool fields on the Operation here

**Comment by madhuravius** [comment]
thanks Mario, my bad - either forgot and/or missed it, maybe we should drop these omitempties by default for our serialization in general (any zero value will get omitted)

**Review by madhuravius — APPROVED** [review]

**Comment by mdelaossa** [comment]
As per Madhu's comments, leaving this un-merged until API/Sweetness changes hit production

**Pull request merged**